client/components/purchase-detail/style.scss

```diff
@@ -152,3 +142,18 @@
 	float: left;
 	height: 14px;
 }
+
+.purchase-detail__button {
+	&[href] {
```

Comment:
Can we target a specific class here? See also https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md